apps/web-portal-client/lib/app/AppContainer.jsx

```diff
@@ -11,14 +11,14 @@ export function AppContainer() {
   const history = useHistory()
   const token = useSelector(selectAuthToken)
 
-  useEffect(() => {
+  /*   useEffect(() => {
```

Comment:
Why this section was commented ?

Reply:
Because auth login doesnt implement and it prevented me to make my tasks

Reply:
How exactly it block you from delivering common components like `Button`, `Image` etc?

Reply:
Aftet submit form everything is ok